components/elm/src/main/elmfates_interfaceMod.F90

```diff
@@ -995,6 +1019,15 @@ subroutine dynamics_driv(this, bounds_clump, top_as_inst,          &
          end if
          this%fates(nc)%bc_in(s)%site_area=col_pp%wtgcell(c)*grc_pp%area(g)*m2_per_km2
 
+         if (use_fates_luh) then
+            ! if (do_landuse_update) then
```

Comment:
Please delete the commented out code and indent the code.

Reply:
done